platform/ethereum/compound/compound.go

```diff
@@ -0,0 +1,216 @@
+package compound
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+)
+
+// Compound Lending Provider
+
+type Provider struct {
+	client Client
+}
+
+func Init(api string) *Provider {
+	return &Provider{
+		client: Client{blockatlas.InitClient(api)},
+	}
+}
+
+const (
+	_providerName                   = "compound"
+	_cacheValiditySec time.Duration = 15 * time.Second
+)
+
+func (p *Provider) Name() string {
+	return _providerName
+}
+
+// GetProviderInfo return static info about the lending provider, such as name and asset classes supported.
+func (p *Provider) GetProviderInfo() (blockatlas.LendingProvider, error) {
+	prov := blockatlas.LendingProvider{
```

Comment:
why call `prov` and then use `assets`. Can you stick to one naming style?

`provider` 
`assets` 

both are easy to use

Reply:
ok